app/uk/gov/hmrc/ngrloginregisterfrontend/models/Address.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ngrloginregisterfrontend.models
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class Address(line1: String,
+                   line2: Option[String],
+                   town: String,
+                   county: Option[String],
+                   postcode: Postcode,
+                   country: String) {
```

Comment:
I think we should add a default to country, "GB" from ISO-3166. . The situation regarding international addresses is not yet resolved but this will give us a bit of flexibility.